site/daemon.md

```diff
@@ -60,16 +60,16 @@ fluxd requires setup and offers customization though a multitude of flags.
 |--git-label             |                               | label to keep track of sync progress; overrides both --git-sync-tag and --git-notes-ref|
 |--git-sync-tag          | `flux-sync`             | tag to use to mark sync progress for this cluster (old config, still used if --git-label is not supplied)|
 |--git-notes-ref         | `flux`            | ref to use for keeping commit annotations in git notes|
-|--git-poll-interval     | `5 minutes`                 | period at which to fetch any new commits from the git repo |
-|--git-timeout           | `20 seconds`                | duration after which git operations time out |
+|--git-poll-interval     | `5m`                 | period at which to fetch any new commits from the git repo |
```

Comment:
Well caught, _en passant_

Reply:
Just noticed there are some more inconsistencies in the docs (https://github.com/weaveworks/flux/pull/1565/files#diff-91111368d299d9de710705677c1ad0f6R28 is not a valid flag, same goes for https://github.com/weaveworks/flux/pull/1565/files#diff-91111368d299d9de710705677c1ad0f6L33) but I will address them in an additional PR.